### ISSUE TITLE
Briefcase prints SLF4J messages when run from the CLI

### ISSUE THREAD

**Issue by yanokwa** [opendatakit/briefcase#168]
#### Software versions 
Briefcase v1.7.0-20-gbb76756

#### Problem description
When I run Briefcase with the CLI, it prints out a logging message before executing the command.
```
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
```

#### Steps to reproduce the problem
$ ./gradlew clean jar;
$ java -jar build/libs/ODK\ Briefcase\ v1.7.0-20-gbb76756.jar -h;

#### Expected behavior
Information message should only be printed when they are asked for.

**Comment by yanokwa** [comment]
@opendatakit-bot claim

**Comment by yanokwa** [comment]
`./gradlew dependencies` says that we have an slf4j dependency because of google-analytics-java:1.1.2. I think that's likely causing the rest of the logging to fail.

```
+--- org.bouncycastle:bcprov-jdk16:1.46
+--- commons-cli:commons-cli:1.3.1
+--- commons-io:commons-io:2.5
+--- org.bushe:eventbus:1.4
+--- org.apache.httpcomponents:httpmime:4.5.2
|    \--- org.apache.httpcomponents:httpclient:4.5.2
|         +--- org.apache.httpcomponents:httpcore:4.4.4
|         +--- commons-logging:commons-logging:1.2
|         \--- commons-codec:commons-codec:1.9
+--- net.sf.kxml:kxml2:2.3.0
+--- org.opendatakit:opendatakit-javarosa:2.5.1
|    +--- net.sf.kxml:kxml2:2.3.0
|    \--- joda-time:joda-time:2.9.9
+--- org.hsqldb:hsqldb:2.4.0
\--- com.brsanthu:google-analytics-java:1.1.2
     +--- org.slf4j:slf4j-api:1.7.5
     +--- org.slf4j:jcl-over-slf4j:1.7.5
     |    \--- org.slf4j:slf4j-api:1.7.5
     \--- org.apache.httpcomponents:httpclient:4.3 -> 4.5.2 (*)
```

**Comment by lognaturel** [comment]
Aha, that makes sense. 

I doubt it's making the rest of the logging fail, though. Any evidence of that?

**Comment by lognaturel** [comment]
Did you try configuring your logging implementation? A bunch of those things used to be printlns.

**Comment by lognaturel** [comment]
Conversation about logging continued in Slack and then #196 and #197 filed.

The slf4j dependency introduced by `google-analytics-java` is indeed a problem. I've reopened #194 because since we don't use SLF4J in core Briefcase it's safe to suppress entirely. If we do switch to SLF4J then I presume a real provider will need be included.

From https://www.slf4j.org/codes.html#StaticLoggerBinder:
> Note that embedded components such as libraries or frameworks should not declare a dependency on any SLF4J binding but only depend on slf4j-api. When a library declares a compile-time dependency on a SLF4J binding, it imposes that binding on the end-user, thus negating SLF4J's purpose.

**Comment by yanokwa** [comment]
Using slf4j-nop suppresses all logging.
Using slf4j-simple prints out all logging, but in a format that suggests it's using slf4j.

Using either with configured logging as described at https://github.com/opendatakit/briefcase/issues/196#issuecomment-328987991 doesn't change this behavior. There isn't a briefcase.log and I see the same output from above on the console.

**Comment by lognaturel** [comment]
Sigh. What if you comment out the rogue dependency and code that uses it? `jcl-over-slf4j` is capturing everything, maybe?

**Comment by NaderVIP** [comment]
yaw you are right about slf4j-nop.jar it's silently discarding all logging.

we can solve it by adding slf4j-jdk14-1.7.25.jar and slf4j-api-1.7.25.jar as a Referenced Libraries not this will make it work at run time. i tried it and when i run the briefcase i get this in console window :

Sep 21, 2017 2:57:22 AM com.brsanthu.googleanalytics.GoogleAnalytics <init>
INFO: Initializing Google Analytics with config=GoogleAnalyticsConfig....................
Sep 21, 2017 2:57:52 AM org.opendatakit.briefcase.model.TerminationFuture markAsCancelled
INFO: cancel requested: User cancels connection.
Sep 21, 2017 2:58:18 AM org.opendatakit.briefcase.model.TerminationFuture markAsCancelled
INFO: cancel requested: Main window closed
Sep 21, 2017 2:58:18 AM org.opendatakit.briefcase.model.TerminationFuture markAsCancelled
INFO: cancel requested: Main window closed

**Comment by lognaturel** [comment]
Closed by #204
